modules/custom/activity_creator/src/ActivityFactory.php

```diff
@@ -103,11 +118,19 @@ private function getFieldMessage($data) {
   /**
    * Get field value for 'output_text' field from data array.
    */
-  private function getFieldOutputText(Message $message) {
+  private function getFieldOutputText(Message $message, $arguments = []) {
     $value = NULL;
     if (isset($message)) {
       $value = $message->getText(NULL);
-      $text = reset($value);
+      // Text for aggregated activities.
+      if (!empty($value[1])) {
+        $text = t($value[1], $arguments);
```

Comment:
will this t() actually work? 

https://www.drupal.org/developing/api/8/localization 
Chapter: Variables in user interface text

Seems like we need literal strings there coming from yml files, instead of message::load and the count parsed in it.